src/patternfly/components/Skeleton/examples/Skeleton.md

```diff
@@ -11,12 +11,14 @@ import './Skeleton.css'
 ### Default
 ```hbs
 {{#> skeleton}}
+    <span class="pf-v6-screen-reader">Loading default content</span>
```

Comment:
There is a hbs partial that will generate this element for you - [`screen-reader.hbs`](https://github.com/patternfly/patternfly/blob/main/src/patternfly/utilities/Accessibility/screen-reader.hbs)

```suggestion
    {{#> screen-reader}}Loading default content{{/screen-reader}}
```